tasks/rh-sign-image/rh-sign-image.yaml

```diff
@@ -63,6 +63,14 @@ spec:
             referenceContainerImage=$(jq -r ".components[${COMPONENTS_INDEX}].containerImage" ${SNAPSHOT_PATH})
 
             reference=$(jq -r ".components[${COMPONENTS_INDEX}].repository" ${SNAPSHOT_PATH})
+
```

Comment:
I am wondering if we should add this as a function in the utils image. Not required for now, but something to think about.

Reply:
Yeah, I agree it sucks that we have this hardcoded in several places now. But it won't be easy to centralize this in one function, because it can be slightly different depending on the context - in Pyxis, everything is translated to registry.access.redhat.com - that applies for both quay.io/redhat-prod and quay.io/redhat-pending. Whereas here, quay.io/redhat-prod translates to registry.redhat.io and quay.io/redhat-pending translates to registry.stage.redhat.io.